Ricochet/Assets/_Art/Chris C Particle Effects.meta

```diff
@@ -0,0 +1,10 @@
+fileFormatVersion: 2
```

Comment:
Can you remove this meta file? The asset doesn't exist, so it keeps getting deleted and readded across branches.